### ISSUE TITLE
Implement lesson cards for app-react

### ISSUE THREAD

**Issue by dalegacusan** [jrobind/dev-jot#133]
Noticed that the **LessonCard** component in *app-react* is a *work under progress*. This should be implemented as soon as possible since it's one of the bare-bone features of the application.

**Comment by dalegacusan** [comment]
The styles of the cards should be the same as the ones from *app-main* right?

**Comment by jrobind** [comment]
Thanks for raising this issue! Totally agree and to answer your question, for now, yes. Let's keep things consistent 👍 

If you want to take this on Dale, I can assign it to you?

**Comment by dalegacusan** [comment]
Definitely! You can assign this to me. I'll probably need to pass #129 so that I can focus on this one.

**Comment by jrobind** [comment]
No worries and no rush on either of them!
